### PR TITLE
295c73

### LINKED ISSUE #2013: 295c73
GDoc: 

Category: NGS library prep

### PR DESCRIPTION
**Pull request by StevenPlonk**
close #2013

### CODE REVIEW

File: protocols/295c73/295c73.ot2.apiv2.py [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+    indexing_sample = tc_plate.wells()[
+      indexing_sample_col*column_well_count:sample_count +
+      indexing_sample_col*column_well_count]
+    output_sample = tc_plate.wells()[
+      output_sample_col*column_well_count:sample_count +
+      output_sample_col*column_well_count]
+    multiplex_mm = temp_reagents.wells_by_name()[mm_well]
+    index = ctx.load_labware("biorad_96_wellplate_200ul_pcr", 4)
+    reservoir = ctx.load_labware("nest_12_reservoir_15ml", '2')
+    ETOH = reservoir.wells()[0]
```
Comment (ncdiehl11) [inline protocols/295c73/295c73.ot2.apiv2.py:75]: generally, we reserve all capital variables for global variables. Let's change this to `etoh`

File: protocols/295c73/fields.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    "label": "Sample Count",
+    "name": "sample_count",
+    "default": 24
+  },
+  {
+    "type": "dropDown",
+    "label": "Hold PCR plate on ice until block reaches 98 degrees?",
+    "name": "hold_pcr_plate_on_ice_until_block_reaches_98",
+    "options": [
+      { "label": "YES", "value": "1" },
```
Comment (ncdiehl11) [inline protocols/295c73/fields.json:13]: we can use the JSON values for booleans here, and not have to typecast them in the Python script. So, we can do:
```
{ "label": "YES", "value": true },
{ "label": "NO", "value": false }
```

File: protocols/295c73/295c73.ot2.apiv2.py [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+    p20.flow_rate.blow_out = 1000
+    p20.transfer(
+      mm_volume, multiplex_mm, initial_sample, mix_after=(4, 15),
+      new_tip='always', disposal_volume=0)
+
+    # deactivate temp deck for later use with room temp cleanup reagents
+    temp.deactivate()
+
+    # pcr
+    if int(hold_pcr_plate_on_ice_until_block_reaches_98):
```
Comment (ncdiehl11) [inline protocols/295c73/295c73.ot2.apiv2.py:107]: per my comment on `fields.json`, we can treat this variable as a boolean, and use:
```
if hold_pcr_plate_on_ice_until_block_reaches_98:
```

**Review by ncdiehl11 — CHANGES_REQUESTED** [review]
Hey Steve, looks good, but I left a few (mostly stylistic) comments.

**Review by chazmanc — APPROVED** [review]

**Comment by chazmanc** [comment]
closes #2014

**Comment by ncdiehl11** [comment]
changes reflected and merged in #2105

**Pull request closed without merging**